crates/bevy_text/src/lib.rs

```diff
@@ -28,7 +28,7 @@ pub mod prelude {
 
 use bevy_app::prelude::*;
 use bevy_asset::AddAsset;
-use bevy_ecs::{entity::Entity, schedule::ParallelSystemDescriptorCoercion};
+use bevy_ecs::{entity::Entity, prelude::Resource, schedule::ParallelSystemDescriptorCoercion};
```

Comment:
nit: avoid importing from prelude and directly import resolved module maybe?

Reply:
suggested changes made